application/configuration.js

```diff
@@ -66,6 +66,20 @@ if (process.env.NODE_ENV === 'test') {
   slidewikiDbName = 'slidewiki_test';
 }
 
+//LTI stuff
```

Comment:
Why did you re-added these obsolete lines???

Reply:
Deleted. Sorry, the local file was not updated.